pages/linux/brightnessctl.md

```diff
@@ -0,0 +1,24 @@
+# brightnessctl
+
+> Utility for reading and controlling device brightness for GNU/Linux operating systems.
+> More information: <https://github.com/Hummer12007/brightnessctl>.
+
+- List devices with available brightness control:
+
+`brightnessctl -l`
+
+- Get the currently set brightness of the device:
+
+`brightnessctl get`
+
+- Set brightness to certain percent of the maximum (eg. 50%):
```

Comment:
```suggestion
- Set the brightness of the display backlight to certain percent of the maximum:
```
I think we could also remove the last part, because you kind of already expect that behaviour
```suggestion
- Set the brightness of the display backlight:
```